src/screens/LandingScreen.js

```diff
@@ -29,22 +31,18 @@ const LandingScreen = ({ navigation }) => {
 		navigation.replace(NAVIGATORS.MAIN);
 	};
 
-	const onAuthStateChanged = async (user) => {
-		if (user) {
-			await loadInitialPlaylists();
-		} else {
-			Alert.alert("Something went wrong please try again");
-		}
+	const handleAnonymousSignIn = async () => {
+		const anonymousUser = await auth().signInAnonymously();
+		dispatch(setUser(anonymousUser));
```

Comment:
you don't want to set the user from the anonymous user, you only want to set it when a user logs in, other wise, we don't use the user